config.go

```diff
@@ -175,9 +175,9 @@ const (
 	defaultRemoteMaxHtlcs = 483
 
 	// defaultMaxLocalCSVDelay is the maximum delay we accept on our
-	// commitment output.
-	// TODO(halseth): find a more scientific choice of value.
-	defaultMaxLocalCSVDelay = 10000
+	// commitment output. The default of 2016 makes it symetric to our
+	// default max csv delay for the remote peer.
```

Comment:
what do you mean by saying being `symetric`?

Reply:
```suggestion
         //The local csv delay maximum is now equal to
	// the remote csv delay maximum we require for the remote commitment
	// transaction.```

Reply:
I meant that the maximum for  remote csv delay and local csv delay are now the same